src/forms/CurrentExpenses.js

```diff
@@ -168,20 +161,22 @@ const ShelterRadio = function ({ currentValue, label, time, setClientProperty })
 };  // End ShelterRadio(<>)
 
 
-/** @todo description
-* 
-* @function
-* @param {object} props
-* @property {object} props.__ - explanation
-* 
-* @returns Component
-*/
+/** 
+ * @function
+ * @param {object} props
+ * @property {object} props.current - Client data of current user circumstances
+ * @property {string} props.type - 'expense' or 'income', etc., for classes
+ * @property {string} props.time - 'current' or 'future'
+ * @property {function} props.setClientProperty - Sets state values
```

Comment:
`@property` here refers not to properties on the parameter, `prop`, but on the function `Housing()`. So you are suggesting that `typeof Housing.props.time === 'string'`.

Granted this usage makes sense to people, so if that's all your intending to use it for, it's fine. But if you ever wanted to generate documentation from this, a computer is going to misunderstand you.